source/_patterns/03-templates/body/body.twig

```diff
@@ -0,0 +1,36 @@
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([]) }) -%}
+
+<body {%- include('atoms-attributes') -%}>
```

Comment:
Thinking that the contents of the `<body>` should be controlled by the Pattern Library, but not the `<body>` itself. 

Imaging things like Google Tag Manager's `<noscript>` which also need to go inside.